include/quicr/detail/tick_service.h

```diff
@@ -104,7 +104,7 @@ namespace quicr {
         std::atomic<bool> stop_{ false };
 
         /// Sleep delay in microseconds
-        const uint64_t sleep_delay_us_{ 10 };
+        const uint64_t sleep_delay_us_{ 333 };
```

Comment:
can we make it a config parameter ? 333ms might to be too slow

Reply:
It's microseconds. Based on our usage now and longer term and considering that sleep relies on interrupts that will result in larger than sleep time variations, the change to 1/3 of a millisecond should be sufficient for our needs.  This provides granularity within ~2/3 of a millisecond.   

The original intent is that we would have 10 microsecond granularity, but that isn't working with sleep or select.  Later we'll revisit the tick service implementation to provide 10us granularity that hopefully avoids the interrupt/yield problem without incurring a busy loop that drives up CPU.